packages/optimizer/spec/transformers/valid/ReorderHeadTransformer/supports_esm/input.html

```diff
@@ -0,0 +1,19 @@
+<!doctype html>
+<html ⚡>
+<head>
+  <script async nomodule src="https://cdn.ampproject.org/v0.js"></script>
```

Comment:
Big problem: The `nomodule` v0.js script is getting dropped in the output. Here's one thing I had to do in the PHP version to account for this: https://github.com/ampproject/amp-toolbox-php/pull/130/commits/4bad718a538c317168a9cb36728a7d977e25b7db

Reply:
fixed